WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListViewModel.swift

```diff
@@ -120,22 +155,38 @@ private extension CustomFieldsListViewModel {
         } else {
             addedFields.append(field)
         }
-
-        updateCombinedList()
     }
 
-    func updateCombinedList() {
-        combinedList = originalCustomFields
-            .filter { field in !deletedFieldIds.contains(where: { $0 == field.id }) }
-            .map { field in editedFields.first(where: { $0.fieldId == field.id }) ?? CustomFieldUI(customField: field) }
-            + addedFields
-    }
+    func observePendingChanges() {
+        $pendingChanges
+            .map { [weak self] pendingChanges in
+                guard let self = self else { return [] }
+
+                return self.originalCustomFields
```

Comment:
Super nit: with the latest version of Swift, we can write as follows:

```suggestion
                guard let self else { return [] }

                return originalCustomFields
```

Reply:
Not needed anymore with the change to make `originalCustomFields` reactive.